charts/kubernetes-event-exporter/values.yaml

```diff
@@ -7,8 +7,6 @@ replicaCount: 1
 image:
   repository: ghcr.io/resmoio/kubernetes-event-exporter
   pullPolicy: IfNotPresent
-  # Overrides the image tag whose default is the chart appVersion.
-  tag: latest
```

Comment:
There are also many charts using empty string "" which is the same as `latest`.

The `latest` ensures users always use the latest image.

If users want to customize the image tag, they could define their own value file.

Reply:
Using latest is the worst way. This will mean you as developers are never allowed to make breaking changes as else you will break everyone's deployment he moment you release it.

I am now using the bitnami chart.